utils/copy-env-vars.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+folders=("dtfs-central-api" "external-api" "gef-ui" "portal" "portal-api" "secrets" "trade-finance-manager-api" "trade-finance-manager-ui" "utils/mock-data-loader")
```

Comment:
I believe we now run all our utils commands from `utils`, so we should copy `.env` to `utils` instead of `utils/mock-data-loader`

Reply:
what about npm run load? Why have we done this out of interest as it adds an extra step during setup/reloading

Reply:
`npm run load` runs from all of `.`, `./utils`, `./utils/mock-data-loader`. I think Abhi commonised our utils `package.json` so now all the utils can be run just from`./utils`

Reply:
What extra step has been added by making this change?

Reply:
misread what you meant by that -- thought you meant we as devs were meant to run in utils. updated